amundsen_application/static/js/ducks/bookmark/tests/index.spec.ts

```diff
@@ -0,0 +1,308 @@
+import { expectSaga, testSaga } from 'redux-saga-test-plan';
+import * as matchers from 'redux-saga-test-plan/matchers';
+import { throwError } from 'redux-saga-test-plan/providers';
+
+import { Bookmark, ResourceType } from 'interfaces';
+
+import { addBookmark as addBkmrk, getBookmarks as getBkmrks, removeBookmark as removeBkmrk } from '../api/v0';
```

Comment:
Seems like we have some naming collisions here. Why not use `addBookmarkAPI`?

Reply:
No solid reason, I just realized re-naming too many things would further obscure the actual content/intent of the PR so I opted for using an alias in the one file where there was a naming collision.